3478/1/top_gems/terminal_output.rb

```diff
@@ -0,0 +1,35 @@
+require 'terminal-table'
+
+class TerminalOutput
+  HEADER = %w[Name Used_by Watchers Stars Forks Contributors Issues].freeze
+
+  def initialize(gem_top, top_size, contains_in_name)
+    @gem_top = gem_top.sort_by { |gem| gem[:total_score] }
```

Comment:
`@gem_top = gem_top.sort_by { |gem| - gem[:total_score] }`

Reply:
do not put logic into initialize

Reply:
What's the differences? In which way is it better?

> `@gem_top = gem_top.sort_by { |gem| - gem[:total_score] }`

Reply:
ok, got it